app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -79,7 +81,10 @@ def send_fast_track_engineer_email_for_testing(form526_submission_id, error_mess
     end
 
     def get_icn(form526_submission)
-      account(form526_submission).icn.presence
+      account = account(form526_submission)
```

Comment:
I think this method would be more readable if the local variable and the method had different names.